### PR TITLE
us001 dashboard feature test added

### PR DESCRIPTION
**Pull request by NesliUsta**
This pull request involves only below pages:
-DashboardPage.java
-Dashboard_StepDef.java
-us001_dashboard.feature

****IMPORTANT NOTE:**** 
To be able to execute above codes in group project framework, 
the url key in LINE 22 inside LoginPage_StepDefinitions.java
must be changed as below:
Driver.getDriver().get(ConfigurationReader.getProperty("url")); 
according to the group project configurations.properties file.

### CODE REVIEW

File: src/test/resources/features/us001_dashboard.feature
Comment (Shohruh0312) [inline src/test/resources/features/us001_dashboard.feature:1]: looks good

**Review by Shohruh0312 — APPROVED** [review]
good job

**Comment by Kelyxa** [comment]
looks good!

File: src/test/resources/features/us001_dashboard.feature
Comment (Kelyxa) [reply src/test/resources/features/us001_dashboard.feature:1]: good job!

**Review by Kelyxa — APPROVED** [review]

File: src/test/resources/features/us001_dashboard.feature
Comment (mery666-ui) [reply src/test/resources/features/us001_dashboard.feature:1]: Nice script,
Good job

**Review by mery666-ui — APPROVED** [review]

**Review by Haden-Z — APPROVED** [review]

**Pull request merged**